engineering/project-template/README.md

```diff
@@ -0,0 +1,98 @@
+![logo if any exists](path/to/logo)
+
+```
+A row of badges you might consider necessary.
+Examples include:
+
+- godoc reference:
+
+    [![GoDoc](https://godoc.org/gopkg.in/src-d/go-git.v4?status.svg)](https://godoc.org/github.com/src-d/go-git)
+
+- build status:
+
+    [![Build Status](https://travis-ci.org/src-d/go-git.svg)](https://travis-ci.org/src-d/go-git)
+    [![Build status](https://ci.appveyor.com/api/projects/status/nyidskwifo4py6ub?svg=true)](https://ci.appveyor.com/project/mcuadros/go-git)
+
+- code coverage:
+
+    [![codecov.io](https://codecov.io/github/src-d/go-git/coverage.svg)](https://codecov.io/github/src-d/go-git)
+
+- go report:
+
+    [![Go Report Card](https://goreportcard.com/badge/github.com/src-d/go-git)](https://goreportcard.com/report/github.com/src-d/go-git)
+
+- stability of the project
+
+    ![stable](https://svg-badge.appspot.com/badge/stability/stable?a)
+```
+
+# Project Name
+
+```
+Short description of the project.
+
+Should mention the kind of project this is (library, demo, tool, etc.) and
+the current [stability status](https://www.npmjs.com/package/stability-badges).
+
+Consider adding screencasts or screenshots when they help understand the project.
+GIFs can give a very good in-place representation of the tool in action.
+See [sqle/gitquet](https://github.com/sqle/gitquery) for an example.
+```
+
+# Installation
+
+```
+Short installation guide or link to longer installation instructions.
+This should include a pre-requisites subsection if needed.
+
+Are there Docker images, packages managers (brew, apt, etc), installations scripts?
+```
+
+# Example of utilization
+
+```
+Basic example or link to an `examples` directory with its own `README.md`.
+For libraries it might be some code.
+For tools it might be a quickstart.
+For demos it should explain how to run the demo.
+
+For libraries, the examples should be as close as possible to the code
+the user will eventually write.
+For instance, when writing Go examples, you should write the code so the
+import path appears in the code and any identifier used is qualified
+with the package name.
+```
+
+# Contribute
+
+[Contributions](https://github.com/src-d/{project}/issues) are more than welcome, if you are interested please take a look to
+our [Contributing Guidelines](docs/CONTRIBUTING.md).
+
+# Code of Conduct
+
+All activities under source{d} projects are governed by the [source{d} code of conduct](docs/CODE_OF_CONDUCT.md).
+
+# License
+
+```
+There's a choice between the following three licenses:
+- GPL v3.0, for core parts
+- Apache License 2.0, for libraries/tools that intended to be consumed by third-parties
+- Creative Commons BY-SA 4.0, for projects which are heavy on creative content
+
+You will find more info about what license to choose at: https://github.com/src-d/guide/blob/master/engineering/licensing.md
+```
+
+Apache License Version 2.0, see [LICENSE.md](LICENSE.md).
+
+```
+or
+```
+
+GPL v3.0 or later, see [LICENSE.md](LICENSE.md).
```

Comment:
just to confirm it: `or later` is not suggesting us to choose a newer version; it is the text we must use (because newer GPL licenses will be also GPL3 compatible)

Reply:
many thanks for the explanation!!!